base/dcp_sharded.go

```diff
@@ -54,6 +54,8 @@ type CbgtContext struct {
 	eventHandlers     *sgMgrEventHandlers      // Event handler callbacks
 	ctx               context.Context          // Log context
 	dbName            string                   // Database name
+	sourceName        string                   // cbgt source name. Store on CbgtContext for access during teardown
```

Comment:
would it be more clear to call this bucketName and bucketUUID? they are only used as sourceName and sourceUUID in cbgt's `CloseStatsClients`

Reply:
They are also sourceName and sourceUUID in c.Manager.CreateIndex.  You're right that for our usage they are always going to be bucketName and bucketUUID, but I thought it would be useful to keep track of them under the name/concept cbgt is using.